tests/common.py

```diff
@@ -330,12 +333,17 @@ class MockModule(object):
     # pylint: disable=invalid-name
     def __init__(self, domain=None, dependencies=None, setup=None,
                  requirements=None, config_schema=None, platform_schema=None,
-                 async_setup=None):
+                 async_setup=None, async_setup_entry=None):
         """Initialize the mock module."""
         self.DOMAIN = domain
         self.DEPENDENCIES = dependencies or []
         self.REQUIREMENTS = requirements or []
-        self._setup = setup
+
+        if setup is None and async_setup is None:
```

Comment:
This fix is saving us a thread per usage of MockModule that did not define `async_setup_entry`!